<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMapLegendSettingEditWidget</class>
 <widget class="QWidget" name="ColorMapLegendSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="visibilityGroupBox">
     <property name="title">
      <string>Visibility</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="visibilityWhenSelectedRadioButton">
        <property name="text">
         <string>When selected</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="visibilityAlwaysRadioButton">
        <property name="text">
         <string>Always</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="visibilityNeverRadioButton">
        <property name="text">
         <string>Never</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>85</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="titleGroupBox">
     <property name="title">
      <string>Title</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
      <item row="0" column="0">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>Title :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="titleFontLabel">
        <property name="text">
         <string>Font Setting:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFontEditWidget" name="titleFontWidget" native="true"/>
      </item>
     </layout>
     <zorder>titleEdit</zorder>
     <zorder>titleLabel</zorder>
     <zorder>titleFontLabel</zorder>
     <zorder>titleFontWidget</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="labelGroupBox">
     <property name="title">
      <string>Label Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="formatLabel">
        <property name="text">
         <string>Format:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="labelFormatEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="numLabelsLabel">
        <property name="text">
         <string>Number of labels:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="numLabelsAutoCheckBox">
          <property name="text">
           <string>Auto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numLabelsSpinBox">
          <property name="minimum">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFontLabel">
        <property name="text">
         <string>Font Setting:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QFontEditWidget" name="labelFontWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorGroupBox">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="titleColorLabel">
        <property name="text">
         <string>Title:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="ColorEditWidget" name="titleColorWidget">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelColorLabel">
        <property name="text">
         <string>Label:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="ColorEditWidget" name="labelColorWidget">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="backgroundColorLabel">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="ColorEditWidget" name="backgroundColorWidget">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TransparencyWidget" name="backgroundOpacitySlider" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="imageSettingGroupBox">
       <property name="title">
        <string>Size and Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="ImageSettingWidget" name="imageSettingWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFontEditWidget</class>
   <extends>QWidget</extends>
   <header>misc/qfonteditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageSettingWidget</class>
   <extends>QWidget</extends>
   <header>guicore/image/imagesettingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>numLabelsAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>numLabelsSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>313</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
